pkg/apis/cluster/v1alpha1/types.go

```diff
@@ -73,6 +73,13 @@ type ClusterSpec struct {
 	// +optional
 	ProxyURL string `json:"proxyURL,omitempty"`
 
+	// ProxyHeader optionally specifies headers to send to
+	// proxy configured in ProxyURL.
+	// The map keys are the HTTP header names, and the values are the associated header payloads.
+	// For the header with multiple values, the map value should be a comma-separated string(e.g. 'Content-Length: 0,1,2')
+	// +optional
```

Comment:
```suggestion
	// ProxyHeader is the HTTP header required by proxy server.
	// The key in the key-value pair is HTTP header key and value is the associated header payloads.
	// For the header with multiple values, the values should be separated by comma(e.g. 'k1': 'v1,v2,v3').
	// +optional
```

Looks good to me. Please talk to @kevin-wangzefeng and @GitHubxsy as well.

Reply:
/lgtm